ml-testing-toolkit/chart-frontend/templates/ingress.yaml

```
@@ -15,7 +15,9 @@ metadata:
       {{ $key }}: {{ $value | quote }}
     {{- end }}
 spec:
-  ingressClassName: nginx
+  {{- if and $.Values.ingress.className (semverCompare ">=1.22-0" $.Capabilities.KubeVersion.GitVersion) }}
+  ingressClassName: {{ default "nginx" $.Values.ingress.className }}
```

Please default it on the values.yaml instead. Otherwise this will "magically" be equal to `nginx` without looking at the actual template code-base. The code is not readable when a helm chart is published, so we should really use the values to default values if that makes sense since it is readable.

```suggestion
  ingressClassName: {{ .Values.ingress.className }}
```

Same comment applies elsewhere.